dev-raku/App-Prove6/metadata.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
1. https, not http
2. Missing any metadata at all, including maintainer?

Reply:
1. I replaced http with https.
2. In my overlay, I don't bother with package medata unless I have to describe USE flags. I had no problem with this approach in my personal overlay. What do you suggest for this?

Reply:
I don't know. Who is going to look after this package? Are you?

Reply:
I think everyone including me. Everyone should take care of gentoo.

Reply:
What should be there if everyone should take the responsibility?

Reply:
I will take care of issues around rakudo.eclass as long as I use raku on gentoo. But, other people should also join.

Reply:
I will just add myself as the maintainer for all affected packages.